### PR TITLE
fix(clickhouse): default logger level set to information

### LINKED ISSUE #701: Clickhouse has a too high log level for production
I took a look at the generated `/var/lib/docker/volumes/sentry_onpremise_sentry-clickhouse-log/_data/clickhouse-server.log` file and I found a lot of `Debug` and `Trace` lines:

```
2020.10.13 15:13:10.712743 [ 55 ] {} <Debug> system.metric_log: Removing part from filesystem 202010_71214_71214_0
2020.10.13 15:13:18.152269 [ 56 ] {} <Trace> SystemLog (system.metric_log): Flushing system log
2020.10.13 15:13:18.156817 [ 56 ] {} <Debug> DiskLocal: Reserving 1.00 MiB on disk `default`, having unreserved 3.62 GiB.
2020.10.13 15:13:18.183674 [ 56 ] {} <Trace> system.metric_log: Renaming temporary part tmp_insert_202010_243_243_0 to 202010_71279_71279_0.
2020.10.13 15:13:25.684530 [ 56 ] {} <Trace> SystemLog (system.metric_log): Flushing system log
2020.10.13 15:13:25.689540 [ 56 ] {} <Debug> DiskLocal: Reserving 1.00 MiB on disk `default`, having unreserved 3.68 GiB.
```

I would like to change the log level, but the `clickhouse/config.xml` is versioned, so I suppose I should not touch it, and nothing is configurable with environment variables.

The log level should be lowed down by default with and an option to change it may be added.

What do you think?

### PR DESCRIPTION
**Pull request by soullivaneuh**
fixes #701

Followed the documentation but did not tested the result.

### CODE REVIEW

File: clickhouse/config.xml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 <yandex>
     <max_server_memory_usage_to_ram_ratio from_env="MAX_MEMORY_USAGE_RATIO" />
+    <logger>
+        <level from_env="CLICKHOUSE_LOGGER_LEVEL" />
+    </logger>
```
Comment (BYK) [inline clickhouse/config.xml:5]:
```suggestion
<logger>
    <level from_env="CLICKHOUSE_LOGGER_LEVEL" />
	<console>1</console>
</logger>
```

File: docker-compose.yml [hunk 2]
```diff
@@ -120,6 +120,7 @@ services:
       # If you have high volume and your search return incomplete results
       # You might want to change this to a higher value (and ensure your host has enough memory)
       MAX_MEMORY_USAGE_RATIO: 0.3
+      CLICKHOUSE_LOGGER_LEVEL: information
```
Comment (BYK) [inline docker-compose.yml:123]:
```suggestion
      CLICKHOUSE_LOGGER_LEVEL: 'information'
```

Let's see if adding quotes around this fixes the issue. If this doesn't fix it, let's just hard-code `information` into the XML file for now. I've Googled this error to no avail.

**Review by BYK — CHANGES_REQUESTED** [review]

**Comment by soullivaneuh** [comment]
@BYK Changed applied and squashed.

**Comment by soullivaneuh** [comment]
Tests are still failing, but I can't identify the real issue on the log. Did you catch it?

**Comment by BYK** [comment]
```
clickhouse_1                   | Poco::Exception. Code: 1000, e.code() = 0, e.displayText() = Invalid argument: Not a valid log level, Stack trace (when copying this message, always include the lines below):
```

This is more informative. Maybe although they document the value as `information`, they actually use `info`? Wanna give that a try? (sorry this turned out to be uglier than it should be)

File: docker-compose.yml [hunk 3]
```diff
@@ -120,6 +120,7 @@ services:
       # If you have high volume and your search return incomplete results
       # You might want to change this to a higher value (and ensure your host has enough memory)
       MAX_MEMORY_USAGE_RATIO: 0.3
+      CLICKHOUSE_LOGGER_LEVEL: 'information'
```
Comment (BYK) [inline docker-compose.yml:123]: Let's try `info`?

```suggestion
      CLICKHOUSE_LOGGER_LEVEL: 'info'
```

File: clickhouse/config.xml [hunk 4]
```diff
@@ -1,3 +1,7 @@
 <yandex>
     <max_server_memory_usage_to_ram_ratio from_env="MAX_MEMORY_USAGE_RATIO" />
+    <logger>
+        <level from_env="CLICKHOUSE_LOGGER_LEVEL" />
```
Comment (BYK) [inline clickhouse/config.xml:4]:
```suggestion
        <level>information</level>
```

File: docker-compose.yml [hunk 5]
```diff
@@ -120,6 +120,7 @@ services:
       # If you have high volume and your search return incomplete results
       # You might want to change this to a higher value (and ensure your host has enough memory)
       MAX_MEMORY_USAGE_RATIO: 0.3
+      CLICKHOUSE_LOGGER_LEVEL: 'info'
```
Comment (BYK) [inline docker-compose.yml:123]:
```suggestion
```

**Review by BYK — COMMENTED** [review]
Let's just try hard-coding.

**Review by BYK — APPROVED** [review]
I guess Clickhouse has a bug for this one. I'll look and see if I can submit a patch upstream but until then, let's go with this hard-coded solution.

**Pull request merged**